components/ui/card.tsx

```diff
@@ -0,0 +1,146 @@
+import * as React from 'react'
+import { cn } from '@/lib/utils'
+import { CardI } from '@/lib/interface/controls/card'
+import Image from 'next/image'
+
+const Card = React.forwardRef<HTMLDivElement, CardI>(
+  (
+    {
+      className,
+      title,
+      children,
+      variant = 'default',
+      primaryColor = 'default',
+      imgUrl,
+      members,
+      subject,
+      desc,
+      Icon,
+      ...props
+    },
+    ref
+  ) => (
+    <>
+      {variant === 'default' && (
+        <div
+          ref={ref}
+          className={cn(
+            'rounded-lg border border-slate-200 bg-white  min-w-[280px] max-w-[300px] h-[160px] text-slate-950 shadow-sm dark:border-slate-800 dark:bg-slate-950 dark:text-slate-50',
+            className
+          )}
+          {...props}
```

Comment:
Don't attach `props` here, it can cause unintended effects if a dev passes a prop you did not expect and that conflicts with your logic.